server/src/companion/settings.py

```diff
@@ -20,70 +20,72 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'c_0@z(0o^520n4gugpe&gdha&ah#%3kk#-t*l$n11n-d5!7rkf'
+SECRET_KEY = "c_0@z(0o^520n4gugpe&gdha&ah#%3kk#-t*l$n11n-d5!7rkf"
```

Comment:
This secret shouldn't have been made public :open_mouth:

Reply:
No worries, these are of no use in our app

Reply:
we are not making any POST or PUT requests, the app is `read-only`